eo-maven-plugin/src/main/java/org/eolang/maven/DcsOf.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2016-2022 Objectionary.com
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package org.eolang.maven;
+
+import java.util.Collection;
+import java.util.Iterator;
+import org.apache.maven.model.Dependency;
+
+/**
+ * Adapter for collection of dependencies.
+ *
+ * @since 0.28.11
+ */
+final class DcsOf implements Dependencies {
```

Comment:
@volodya-lombrozo what this class is for? Where and _why_ `Collection<Dependency>` is not good enough?

Reply:
I wanted to make it more obvious. We can't use it directly in the next construction:
```java
           new DcsNoOneHasTransitive(
                        this.ignoreTransitive,
                        dcs,
                        ....
           )
```
because `Collection<org.apache.maven.model.Dependency>` != `Dependencies`. But, of course, we can use it this way:
```java
           new DcsNoOneHasTransitive(
                        this.ignoreTransitive,
                        dcs::iterator,
                        ....
           )
```
I will fix it. Thank you.

Reply:
@yegor256 fixed.